tests/enzyme.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
Is there a reason you can't put this line inside a regular Meteor/mocha test (run via `practicalmeteor:mocha` etc?)